### PR TITLE
feat: added advanced parameter generation

### PR DESCRIPTION
**Pull request by Jaryt**

### CODE REVIEW

**Review by brivu — APPROVED** [review]
LGTM!

File: .github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml [hunk 1]
```diff
@@ -1,21 +1,25 @@
 name: "\U0001F4A1 Feature Request"
-description: Have an idea for a new feature? Begin by submitting a Feature Request
+description:
```
Comment (KyleTryon) [inline .github/ISSUE_TEMPLATE/FEATURE_REQUEST.yml:2]: Can we pull this into a different commit
